### ISSUE TITLE
Holes are not accounted when calculating surface metric

### ISSUE THREAD

**Issue by arover6** [umap-project/umap#680]
When a hole is made in a surface, the metric that is calculated by right-clicking is not accounting the hole, this can bias a lot the result (and is seemingly an unintended effect).

This was tested on the version released on the website.

**Comment by Binnette** [comment]
Hi @arover6, sorry i do not understand your issue. Can you please add a screenshot and the steps to reproduce this issue? Also, it may be an Leaflet issue more than a uMap issue, if so please consider filling a bug to Leaflet.

**Comment by yohanboniface** [comment]
I guess it's an issue with https://github.com/umap-project/Leaflet.Measurable

Polygons can contain holes, and certainly the area computation does not account them.

**Comment by arover6** [comment]
@yohanboniface Yes exactly this issue. It's weird that the holes are not accounted for, if it's too complex mathematically, at least maybe a warning could be added (that the measure won't be accurate if there are holes in the polygon)?
